types/index.d.ts

```diff
@@ -38,6 +38,18 @@ declare module '@reach/menu-button' {
 
   export const MenuList: React.SFC<MenuListProps>;
 
+  export type Position = (
+    targetRect?: PRect | null,
+    popoverRect?: PRect | null,
+  ) => React.CSSProperties;
+
+  export type MenuPopoverProps = React.HTMLAttributes<HTMLDivElement> & {
+    children: React.ReactNode;
+    position?: Position;
+  };
+
+  export const MenuPopover: React.SFC<MenuPopoverProps>;
+
```

Comment:
Was these some custom types or are those just missing from the the existing [@types/reach__menu-button](https://www.npmjs.com/package/@types/reach__menu-button) _(seems to be 8 months old...)_ ?

Reply:
These are copied from the library. We do have some custom stuff here in the reach module definition and I'm going to create a following ticket to ditch this completely if possible. Just did not want to mix imported reach types with our custom definition due to conflict risk.